### DISCUSSION TITLE
来点图图

### DISCUSSION THREAD

**Discussion by Furina-deFontaine** [share121/inter-knot#4977] [其他]
![Uploading 120451603_p0.png…]()

**Comment by Furina-deFontaine** [comment]
好像发不出来图片？

**Comment by HLXS-HLXS** [comment]
发图时要等一等，等图片那串数字后面出现了网址再发送，网址出现后图片才上传完
![112755914_p0](https://github.com/user-attachments/assets/b58cd151-d7bc-477f-90b2-79ed09030960)
![112333531_p0](https://github.com/user-attachments/assets/0cba7a3b-b9fd-4b8d-b71b-be2b0fdc032c)

**Reply by Furina-deFontaine** [reply]
好的好的，感谢😘

**Comment by xxx071220** [comment]
只能发网址上的图片？

**Comment by Carkys** [comment]
![image](https://github.com/user-attachments/assets/cc8f4b2c-73c9-4c77-9af2-eb90c8bb067c)
